### PR TITLE
Adds several negatives traits back to genemodder

### PR DESCRIPTION
**Pull request by Guidesu**
Hello.
I am adding every negative trait back to genemodders.
Why you may ask...
Because only having 5 traits sucks.
And only negatives ones to the FBP genemoders don't power-game again.
The only positive trait i added back was telepathy, wich is not that game-breaking at all.

### CODE REVIEW

**Comment by gilvian** [comment]
really dont. we took them out for a reason

We will not be adding in negative and posivite traits in a near future or possibly even at all.

**Comment by Guidesu** [comment]
I just talked with Allani.
He is just fine with it.

"I asked peek if he can fix genemodders
to prevent IPC from being selectable and such
this way we can fix traits, make them pos/neg actually work and such"

And you toke they out because of Powergaming FBP
Wich, you nuked all my characters
I don't see a reason to not.

**Comment by Guidesu** [comment]
So your statement
"We will not be adding in negative and posivite traits in a near future or possibly even at all."
Is cucked up
The host is planning to add it back.

**Comment by gilvian** [comment]
Shes a host, we are devs. In the end we really have a saying going "hey no thats a bit too dumb" and Alanii respects it most of the time. 

While yeah "its non combat positive trait" telepathy was not a broken, abused mess on old Hestia that lead people into trying to get it disabled few times and people noted and warned about using it.

**Comment by Guidesu** [comment]
So just tell me which traits are broken to remove.
Its kinda simple

**Comment by Guidesu** [comment]
Because.
I don't see how any of these traits are broken
They will make your game harder.
Because if you want to handicap yourself.
Making you take more brute or burns
That up to yourself.
Because how that is game breaking?
Tell me.

**Comment by Guidesu** [comment]
And ALSO tell me.
Why literally limit some people because of power gaming ones?
It's also the SAME discussion about Slime people.
Some people were BADLY power gaming then you just cucked up nerfing they instead of fixing it

File: modular_mithra/code/modules/mithra/traits_neutral.dm [hunk 1]
```diff
@@ -58,4 +76,92 @@
...
+	name = "Incredibly Frail"
+	desc = "Your bones, skin and general state of mind is rather fragile. Try not to get smacked, or you may have to visit the ER."
+	var_changes = list("brute_mod" =2.50)
+
+	/datum/trait/commune
+	name = "Telepathy"
+	desc = "Quite simply, you've the ability to project thoughts into the minds of others. A weak psychic manifestation too minor to require action from the local authorities, unlikely to ever develop into something greater."
+/datum/trait/commune/apply(var/datum/species/S,var/mob/living/carbon/human/H)
+	..(S,H)
+	H.verbs |= /mob/living/carbon/human/proc/psychic_whisper
```
Comment (io-util) [inline modular_mithra/code/modules/mithra/traits_neutral.dm]: We talked about that, there's no purpose for it's existance other than abuse, it was abused in the past and really adds nothing positive.

**Review by io-util — CHANGES_REQUESTED** [review]

**Comment by Guidesu** [comment]
So please don't come with this discussion on top of me.
I am not changing the game in several ways.
I am just bringing old /NEGATIVES/I repeat NEGATIVES ones.
How this can be bad?

File: modular_mithra/code/modules/mithra/traits_neutral.dm (hunk 1 above)
Comment (Guidesu) [reply modular_mithra/code/modules/mithra/traits_neutral.dm]: right

**Comment by io-util** [comment]
> So please don't come with this discussion on top of me.
> I am not changing the game in several ways.
> I am just bringing old /NEGATIVES/I repeat NEGATIVES ones.
> How this can be bad?

Telepathy totally is not "negative" and has no positive use, it's a dumb trait.

**Comment by Guidesu** [comment]
hmm okay then.
I understand your reason

**Comment by Guidesu** [comment]
i think i commited the changes

**Comment by Guidesu** [comment]
uhm...
Can you review?

File: modular_mithra/code/modules/mithra/traits_neutral.dm [hunk 2]
```diff
@@ -58,4 +76,84 @@
...
+
+/datum/trait/hemophilia
+	name = "Hemophilia"
+	desc = "Your body doesn't quite stop bleeding once it starts. You need immediate treatment for anything, even minor wounds, or it might turn out real bad for you."
+	var_changes = list("blood_volume" = 105) //The bare minimum before it becomes critical.
+
+/datum/trait/stick_human
+	name = "Incredibly Frail"
+	desc = "Your bones, skin and general state of mind is rather fragile. Try not to get smacked, or you may have to visit the ER."
+	var_changes = list("brute_mod" =2.50)
```
Comment (io-util) [inline modular_mithra/code/modules/mithra/traits_neutral.dm]: Just a cosmetic thing but move the `=`.

```suggestion
	var_changes = list("brute_mod" = 2.50)
```

**Review by io-util — CHANGES_REQUESTED** [review]

**Comment by Guidesu** [comment]
done

File: modular_mithra/code/modules/mithra/traits_neutral.dm [hunk 3]
```diff
@@ -58,4 +76,83 @@
...
+
+/datum/trait/speed_slow
+	name = "Sluggish"
+	desc = "You move slower than the average person."
+	var_changes = list("slowdown" = 1.5)
+
+/datum/trait/endurance_low
+	name = "Fragile"
+	desc = "Your body is much, much more fragile than the average joe."
+	var_changes = list("total_health" = 65)
```
Comment (Firefox13) [inline modular_mithra/code/modules/mithra/traits_neutral.dm]: but default `total_health` equals to 200? are you sure?
изображение

Comment (Guidesu) [reply modular_mithra/code/modules/mithra/traits_neutral.dm]: I didint modified that at all.
This was in the old vesta the entire time

Comment (io-util) [reply modular_mithra/code/modules/mithra/traits_neutral.dm]: Up that a little.

Comment (Guidesu) [reply modular_mithra/code/modules/mithra/traits_neutral.dm]: done

**Comment by io-util** [comment]
Closing this as we are not reviewing / accepting PR's on our old repo anymore.
New Repo: https://github.com/VestaOfOrion/Vesta.Bay/

**Pull request closed without merging**